python_transport/wirepas_gateway/transport_service.py

```diff
@@ -376,6 +377,24 @@ def on_stack_started(self, name):
         topic = TopicGenerator.make_set_config_response_topic(self.gw_id, sink.sink_id)
         self.mqtt_wrapper.publish(topic, response.payload, qos=2)
 
+    def on_stack_stopped(self, name):
+        self.logger.debug("Sink stopped: %s", name)
+        sink = self.sink_manager.get_sink(name)
+        if sink is None:
+            self.logger.error("Sink stopped %s error: unknown sink", name)
+            return
+
+        # Generate a setconfig answer with req_id of 0
+        response = wmm.SetConfigResponse(
+            0,
+            self.gw_id,
+            wmm.GatewayResultCode.GW_RES_OK,
+            sink.sink_id,
+            sink.read_config(),
+        )
+        topic = TopicGenerator.make_set_config_response_topic(self.gw_id, sink.sink_id)
+        self.mqtt_wrapper.publish(topic, response.payload, qos=2)
+
```

Comment:
This code should be factorize with the one in on_stack_started

Reply:
done